src/robot/utils/robotpath.py

```diff
@@ -25,14 +25,6 @@
 from .unic import unic
 
 
-if sys.version_info < (2,7):
-    def _abspath(path):
-        if WINDOWS and os.path.splitunc(path)[0]:
-            return os.path.abspath(path)
-        return os.path.abspath(os.path.join(os.getcwdu(), path))
-else:
-    _abspath = os.path.abspath
-
 if PY2:
```

Comment:
Since this PR is dropping support for python2.6, can this `if` block be removed?

Reply:
Python 2.7 is still supported (and [CPython until 2020](https://en.wikipedia.org/wiki/CPython#Version_history)).